plugin/ale.vim

```diff
@@ -14,7 +14,7 @@ let g:loaded_ale_dont_use_this_in_other_plugins_please = 1
 
 " A flag for detecting if the required features are set.
 if has('nvim')
-    let s:has_features = has('timers')
+    let s:has_features = has('nvim-0.2.0')
```

Comment:
We can't change this just yet. First we'll need to get version 1.9 out and emit a few more deprecation warnings, and then we can require NeoVim 0.2.x or above. NeoVim versions below that are deprecated for now, but still supported.